projects/objectnav_baselines/experiments/objectnav_mixin_ddppo.py

```diff
@@ -1,9 +1,30 @@
+from typing import Dict, Tuple
 import torch
 import torch.optim as optim
 from torch.optim.lr_scheduler import LambdaLR
 
+from allenact.algorithms.onpolicy_sync.losses.abstract_loss import (
+    AbstractActorCriticLoss,
+)
 from allenact.algorithms.onpolicy_sync.losses import PPO
 from allenact.algorithms.onpolicy_sync.losses.ppo import PPOConfig
+
+from allenact.embodiedai.aux_losses.losses import (
+    MultiAuxTaskNegEntropyLoss,
+    InverseDynamicsLoss,
+    TemporalDistanceLoss,
+    CPCA1Loss,
+    CPCA2Loss,
+    CPCA4Loss,
+    CPCA8Loss,
+    CPCA16Loss,
+)
+from allenact.embodiedai.models.fusion_models import (
+    AverageFusion,
+    SoftmaxFusion,
+    AttentiveFusion,
+)
```

Comment:
Can you add
```python
# noinspection PyUnresolvedReferences
```
just above line 22? This will ensure that we don't accidentally remove this import as it's potentially used below after uncommenting.